docs/dev/check_references.md

```diff
@@ -99,6 +111,76 @@ The complete list of mandatory and optional attributes for the `manifest.json` f
+| `author `                                            | Object                                                                  | Mandatory                                  | Information about the author of this app.                                                                                                                                                                                                                  |   |
+| `author[homepage] `                                  | String (URL)                                                            | Mandatory                                  | The web URL to the homepage of the author.                                                                                                                                                                                                                 |   |
+| `author[name]`                                       | String                                                                  | Mandatory                                  | The human-readable name for this company.                                                                                                                                                                                                                  |   |
+| `author[sales_email]`                                | String (Email)                                                          | Mandatory                                  | The email to contact for any subscription-level events.                                                                                                                                                                                                    |   |
+| `author[support_email]`                              | String (Email)                                                          | Mandatory                                  | The email to contact for any support and maintenance queries.                                                                                                                                                                                                  |   |
+| `author[vendor_id]`                                  | String                                                                  | Mandatory                                  | The vendor ID to use for subscription purposes. Must be globally unique and cannot be changed. Should follow the strict standards of `integration_id` where only dashes and alphabetic chars are allowed. This value is provided to partners. |   |
```

Comment:
Lets change this to reference app_id since V2 doesn't have an `integration_id`

```suggestion
| `author[vendor_id]`                                  | String                                                                  | Mandatory                                  | The vendor ID to use for subscription purposes. Must be globally unique and cannot be changed. Should follow the strict standards of `app_id` where only dashes and alphabetic chars are allowed. This value is provided to partners. |   |
```